kansha/services/search/schema.py

```diff
@@ -32,9 +32,9 @@ class MyDoc(Document):
 class FieldType(object):
 
     parent = None
-    name = ''
 
-    def __init__(self, indexed=True, stored=False, default=None):
+    def __init__(self, name='', indexed=True, stored=False, default=None):
```

Comment:
Personaly, I'd want to ditch it.

But, in the case we keep it, it must be rewrite "on top" of the imperative "Schema" class. By keeping a `schema` attribute or, if possible, perhaps the `Schema` class itself could be both imperative and declarative